great_expectations/data_context/store/expectations_store.py

```diff
@@ -190,7 +225,7 @@ def _add_ids_on_update(self, suite: ExpectationSuite) -> ExpectationSuite:
     def _add_cloud_ids_to_local_suite_and_expectations(
         self, local_suite: ExpectationSuite, cloud_suite: dict
     ) -> ExpectationSuite:
-        if not local_suite.ge_cloud_id:
+        if not local_suite["ge_cloud_id"]:
```

Comment:
Why change this from dotted syntax to a dictionary like key lookup?

Reply:
i think this was because the `CloudMigrator` tests are passing serialized suites to the store instead of deserialized suites - will confirm